pkg/apis/pipeline/v1alpha1/run_types.go

```diff
@@ -90,6 +94,14 @@ const (
 	RunSpecStatusCancelled RunSpecStatus = "RunCancelled"
 )
 
+// RunSpecStatusMessage defines human readable status messages for the TaskRun.
+type RunSpecStatusMessage string
+
+const (
+	// Run cancelled as the Pipeline running it was cancelled.
+	RunCancelledByPipelineMsg RunSpecStatusMessage = "Run cancelled as pipeline has been cancelled."
```

Comment:
I'd also reword this (and the `TaskRun` equivalent) to something like `Run cancelled as the PipelineRun it belongs to has been cancelled.`

Reply:
Done.